### PR TITLE
[Terra] Integration Update

### PR DESCRIPTION
**Pull request by YunSuk-Yeo**
## Description

Update Terra integration interface with protobuf changes and remove unnecessary type field of tx in `cosmos` and `kava`.

## Testing instructions

* ./tools/tests 

## Types of changes

* New feature (non-breaking change which adds functionality)

## Checklist

- [ ] Prefix PR title with `[WIP]` if necessary.
- [ ] Add tests to cover changes as needed.
- [ ] Update documentation as needed.

### CODE REVIEW

**Comment by hewigovens** [comment]
it's safe to remove outer side `type` field?

File: tools/ios-test [hunk 1]
```diff
@@ -7,5 +7,5 @@ set -e
 pushd swift
 xcodegen
 pod install
-xcrun xcodebuild -workspace TrustWalletCore.xcworkspace -scheme TrustWalletCore -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 11,OS=13.0' test
+xcrun xcodebuild -workspace TrustWalletCore.xcworkspace -scheme TrustWalletCore -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 11,OS=13.3' test
```
Comment (hewigovens) [inline tools/ios-test]: CI is still iOS 13.0, please revert this change

File: swift/Tests/Blockchains/TerraTests.swift [hunk 2]
```diff
@@ -9,84 +9,370 @@ import TrustWalletCore
...
+            $0.toAddress = "terra1hdp298kaz0eezpgl6scsykxljrje3667d233ms"
+            $0.amounts = [CosmosAmount.with {
+                $0.amount = 1000000
+                $0.denom = "uluna"
+            }]
+            $0.typePrefix = "bank/MsgSend"
+        }
+
         let message = CosmosMessage.with {
-            $0.rawJsonMessage = CosmosMessage.RawJSON.with {
```
Comment (hewigovens) [inline swift/Tests/Blockchains/TerraTests.swift]: can you add another RawJSON test? we keep it to test arbitrary message

**Comment by YunSuk-Yeo** [comment]
> it's safe to remove outer side `type` field?

yes, actually outer type field is not used. Moreover, default outer type is `cosmos-sdk/StdTx`.

I will add some RawJson test soon

**Comment by hewigovens** [comment]
Android test is failed

**Comment by YunSuk-Yeo** [comment]
Fixed andriod test & append RawJSON test

**Pull request merged**